test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1797,4 +2536,305 @@ static public void createClassWithVolatileValueTypeFields() throws Throwable {
+		Class assortedValueWithSingleAlignment2DClass = Array.newInstance(AssortedValueWithSingleAlignment.class, 1).getClass();
+
+		Object assortedRefWithLongAlignment2DArray = Array.newInstance(assortedValueWithLongAlignment2DClass, genericArraySize);
+		Object assortedRefWithSingleAlignment2DArray = Array.newInstance(assortedValueWithSingleAlignment2DClass, genericArraySize);
+
+		for (int i = 0; i < genericArraySize; i++) {
+			Object ref = Array.get(assortedRefWithLongAlignment2DArray, i);
+			assertNull(ref);
+
+			ref = Array.get(assortedRefWithSingleAlignment2DArray, i);
+			assertNull(ref);
+		}
+	}
+
+	@Test(priority=1)
+	static public void testCyclicalStaticFieldDefaultValues1() throws Throwable {
+		String cycleA1[] = { "val1:LCycleB1;:static" };
+		String cycleB1[] = { "val1:LCycleA1;:static" };
+
+		Class cycleA1Class = ValueTypeGenerator.generateValueClass("CycleA1", cycleA1);
+		Class cycleB1Class = ValueTypeGenerator.generateValueClass("CycleB1", cycleB1);
+
+		MethodHandle makeCycleA1 = lookup.findStatic(cycleA1Class, "makeValueGeneric", MethodType.methodType(Object.class));
+		MethodHandle makeCycleB1 = lookup.findStatic(cycleB1Class, "makeValueGeneric", MethodType.methodType(Object.class));
+
+		makeCycleA1.invoke();
+		makeCycleB1.invoke();
+	}
+
+	@Test(priority=1)
+	static public void testCyclicalStaticFieldDefaultValues2() throws Throwable {
+		String cycleA2[] = { "val1:LCycleB2;:static" };
+		String cycleB2[] = { "val1:LCycleC2;:static" };
+		String cycleC2[] = { "val1:LCycleA2;:static" };
+
+		Class cycleA2Class = ValueTypeGenerator.generateValueClass("CycleA2", cycleA2);
+		Class cycleB2Class = ValueTypeGenerator.generateValueClass("CycleB2", cycleB2);
+		Class cycleC2Class = ValueTypeGenerator.generateValueClass("CycleC2", cycleC2);
+
+		MethodHandle makeCycleA2 = lookup.findStatic(cycleA2Class, "makeValueGeneric", MethodType.methodType(Object.class));
+		MethodHandle makeCycleB2 = lookup.findStatic(cycleB2Class, "makeValueGeneric", MethodType.methodType(Object.class));
+		MethodHandle makeCycleC2 = lookup.findStatic(cycleB2Class, "makeValueGeneric", MethodType.methodType(Object.class));
+
+		makeCycleA2.invoke();
+		makeCycleB2.invoke();
+		makeCycleC2.invoke();
+	}
+
+	@Test(priority=1)
+	static public void testCyclicalStaticFieldDefaultValues3() throws Throwable {
+		String cycleA3[] = { "val1:LCycleA3;:static" };
+
+		Class cycleA3Class = ValueTypeGenerator.generateValueClass("CycleA3", cycleA3);
+
+		MethodHandle makeCycleA3 = lookup.findStatic(cycleA3Class, "makeValueGeneric", MethodType.methodType(Object.class));
+
+		makeCycleA3.invoke();
+	}
+
+	//*******************************************************************************
+	// checkcast tests
+	//*******************************************************************************
+	@Test(priority=1)
+	static public void testCheckCastValueTypeOnInvalidClass() throws Throwable {
```

Comment:
The original `testCheckCastValueTypeOnInvalidLtype()` is using try catch block. Is there a reason the try catch is removed ?

Reply:
Catching the exception itself will give a more clear failure then catching and calling Assert.fail. I seem to recall letting exceptions be thrown organically as a best practice for openj9 tests as well.